src/rgw/rgw_admin.cc

```diff
@@ -558,8 +557,10 @@ static int get_cmd(const char *cmd, const char *prev_cmd, const char *prev_prev_
     if (strcmp(cmd, "list") == 0)
       return OPT_BI_LIST;
   } else if (strcmp(prev_cmd, "period") == 0) {
-    if (strcmp(cmd, "prepare") == 0)
-      return OPT_PERIOD_PREPARE;
+    if (strcmp(cmd, "prepare") == 0) {
+      cerr <<"command is obsolete; please check usage and/or man page" <<std::endl;
+      return 0;
+    }
```

Comment:
@cbodley  Thanks.
I will remove OPT_PERIOD_PREPARE in next patch. 
I think if we remove the message then in next release if user execute period prepare command then he might wonder about period prepare command, means what happened to this command. Should we put any other message or  completely remove this message ("  command is obsolete; please check usage and/or man page")